atomicfu-js/src/main/kotlin/kotlinx/atomicfu/Trace.kt

```diff
@@ -0,0 +1,20 @@
+package kotlinx.atomicfu
+
+public actual class Trace actual constructor (size: Int) {
+    private val mask = size - 1
+    private val a = arrayOfNulls<String>(size)
+    private val index = atomic(0)
+
+    init {
+        check(size and mask == 0) { "size must be power of 2: $size"}
```

Comment:
it's usually worth to round up to the next power of two instead of errors `1 << (32 - Integer.numberOfLeadingZeros(value - 1));`

Reply:
Agree. Let's round up to the next power of two.